site/schemas/create-consequence-stops.schema.ts

```diff
@@ -0,0 +1,130 @@
+import { Severity } from "@create-disruptions-data/shared-ts/enums";
+import { z } from "zod";
+import { setZodDefaultError, zodTimeInMinutes } from "../utils";
+
+export const stopsImpactedSchema = z.object({
+    id: z.number({
```

Comment:
We should only need to define the stop attributes we actually need here, zod will remove any fields that aren't in the schema when we parse it. SIRI needs the commonName, atcoCode, latitude and longitude

Reply:
done